UofT-DSI | sql- Homework 1

ShayanHodai opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Creating diagram to visualize tables relationships

## What did you learn from the changes you have made?
Connecting to database, tables, columns, and understating tables relationships using primary and foreign key

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
N/A

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Identifying tables relationships like many-to-many, one-to-many, etc.

## How were these changes tested?
I tested tables relationships by logic. Like How many products could be in one purchase? And how many purchase can have one product? => many-to-many relationship between purchase and product tables

## A reference to a related issue in your repository (if applicable)
N/A

## Checklist
- [x] I can confirm that my changes are working as intended

ShayanHodai committed:
tables relationship diagram

ShayanHodai committed:
tables relationship diagram

ShayanHodai committed:
Rename tables_diagram.drawio to homework_1.drawio

SidraBushra commented:
Hi Shayan, I cannot see or open your logical model, you need to upload screenshot your model in PNG or JPEG.

ShayanHodai committed:
jpg diagram

ShayanHodai committed:
Merge branch 'homework_1' of https://github.com/ShayanHodai/sql into homework_1

ShayanHodai commented:
> Hi Shayan, I cannot see or open your logical model, you need to upload screenshot your model in PNG or JPEG.

Thanks Sidra, I updated the pull request and added the converted jpg format.

SidraBushra commented:
HW1 marked.
